renderdoc/driver/gl/wrappers/gl_draw_funcs.cpp

```diff
@@ -4445,7 +4445,29 @@ bool WrappedOpenGL::Serialise_glClear(SerialiserType &ser, GLbitfield mask)
       if(mask & (eGL_DEPTH_BUFFER_BIT | eGL_STENCIL_BUFFER_BIT))
         draw.flags |= DrawFlags::ClearDepthStencil;
 
-      AddDrawcall(draw, true);
+      ResourceId res_id;
+      auto recordResourceUses = [&res_id, this](const ContextPair &ctx, GLenum type, GLuint attachment) {
+        if(attachment)
+        {
+          GLResource res = MakeNullResource;
```

Comment:
The default initialiser of `GLResource` also initialises everything to unknown/0, `MakeNullResource` is an implementation detail for the templated resource manager, so you shouldn't try to initialise with that directly.